lib/Fhp/Segment/BME/HIBMESv2.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Fhp\Segment\BME;
+
+use Fhp\Segment\BaseGeschaeftsvorfallparameter;
+use Fhp\Segment\DME\HIDXES;
+use Fhp\Segment\DME\SEPADirectDebitMinimalLeadTimeProvider;
+
+/**
+ * Segment: Terminierte SEPA-Sammellastschrift einreichen Parameter
+ *
+ * @link https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Messages_Geschaeftsvorfaelle_2015-08-07_final_version.pdf
+ * Section: C.10.3.2.2.1 c)
+ */
+class HIBMESv2 extends BaseGeschaeftsvorfallparameter implements HIDXES
```

Comment:
The interface should now be called `HIXXES`.

Reply:
OK, makes sense. Is the file location ok? This is obviously an interface which referenced multiple times and resides in DME.

When querying for the MinimalLead time, this will always result in an object of ..../DME/... regardles what transaction is used. Not sure whether this matters??

Reply:
The file location is okay. There are a couple of such interfaces that are also implemented by segments in other namespaces. It's not very pretty, but better than moving them to the top-level or some unrelated third namespace.

As for the MinimalLead time, I don't know what that is.